docs/csi-volume-import.md

```diff
@@ -98,3 +95,7 @@ spec:
      ports:
      - containerPort: 80
 ```
+4. Clone the imported PVC or take snapshot from it. 
+
+#Known Issue: 
+If the imported PVC has not been used by a pod at step 3, it would fail to clone it or take snapshot.  This is because the volume has not been populated in PSO database. 
```

Comment:
I would not say about the PSO database, but actually say that this is caused by a missing CSI call in the spec that is being investigated by the community

Reply:
It might also be worth detailing the workaround of mounting in a dummy pod.